test/functional/fixtures/tty-test.c

```diff
@@ -6,6 +6,9 @@
 #include <stdlib.h>
 #include <uv.h>
 
+// -V:STRUCT_CAST:641
+#define STRUCT_CAST(Type, obj) ((Type *)(obj))
```

Comment:
Yeah, but I wonder if it could hide something in the future. Maybe we should leave the PVS warnings.